src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -780,9 +780,10 @@ public ResponseEntity<byte[]> exportNetwork(
             @PathVariable("nodeUuid") UUID nodeUuid,
             @PathVariable("format") String format,
             @RequestParam(value = "formatParameters", required = false) String parametersJson,
-            @RequestParam(value = "studyName", required = false) String studyName) {
+            @RequestParam(value = "studyName", required = false) String studyName,
```

Comment:
If the user does not provide a filename, the file should be automatically named NameNode_NameStudyInDirectory. To handle this, the front-end should check if the filename parameter is provided. If it is, there's no need to pass the studyName. If not, the network conversion server should construct the default filename.

Reply:
But if this endpoint is only used in this case : https://github.com/gridsuite/gridstudy-app/pull/2196 fileName will always be used (because the default .value NameNode_NameStudyInDirectory is generated and turned into fileName on the front side)

Actually I may make the _filename_ parameter required.